### PR TITLE
Journal: new feature requests

### PR DESCRIPTION
**Pull request by melisabok**
- Add a filter to author can filter the AEs that are not available or maxed out the quota.
- Add a pre process function to avoid the AE recommendation of AEs that are not available or don't have quota
- Add a button for the EIC to enable the review rating if there is a recommendation missing.
- Create an archived AE group to move the users that are not AEs anymore but they still have assigned papers under review
- Create expert reviewers group and a group invitation that add the "expert reviewer certification" when one of the authors is an expert reviewer.
- fix issue when AE is not in the group and the edge that says unavailable send the reminder

### CODE REVIEW

**Comment by melisabok** [comment]
there are a few pending changes, TMLR wants to show the expert reviewer in the forum page so I'm going to work on that.

**Comment by melisabok** [comment]
it should be ready now!

File: openreview/journal/invitation.py [hunk 1]
```diff
@@ -3043,18 +3046,33 @@ def set_accepted_invitation(self):
             }            
 
         if self.journal.get_certifications():
+            certificaaion_options = self.journal.get_certifications()
```
Comment (celestemartinez) [inline openreview/journal/invitation.py:3049]:
```suggestion
            certification_options = self.journal.get_certifications()
```

File: openreview/journal/invitation.py [hunk 2]
```diff
@@ -3043,18 +3046,33 @@ def set_accepted_invitation(self):
...
+            if self.journal.has_expert_reviewers():
+                certificaaion_options.append(self.journal.get_expert_reviewer_certification())
```
Comment (celestemartinez) [inline openreview/journal/invitation.py:3051]:
```suggestion
                certification_options.append(self.journal.get_expert_reviewer_certification())
```

File: openreview/journal/invitation.py [hunk 3]
```diff
@@ -3043,18 +3046,33 @@ def set_accepted_invitation(self):
...
             invitation.edit['note']['content']['certifications'] = {
                 'order': 3,
                 'description': 'Certifications are meant to highlight particularly notable accepted submissions. Notably, it is through certifications that we make room for more speculative/editorial judgement on the significance and potential for impact of accepted submissions. Certification selection is the responsibility of the AE, however you are asked to submit your recommendation.',
                 'value': {
                     'param': {
                         'type': 'string[]',
-                        'enum': self.journal.get_certifications(),
+                        'enum': certificaaion_options,
```
Comment (celestemartinez) [inline openreview/journal/invitation.py:3058]:
```suggestion
                        'enum': certification_options,
```

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**